web/pandas/about/team.md

```diff
@@ -52,7 +52,15 @@ The project governance is available in the [project governance page]({{ base_url
 ### {{ workgroup.name }}
 
 <ul>
-    <li><b>Contact:</b> <a href="mailto:{{ workgroup.contact }}">{{ workgroup.contact }}</a></li>
+    <li><b>Contact:</b>
+        <a id="{{ ''.join( workgroup.name.split(' ') ) }}" href="mailto:{{ 'asp.' + workgroup.contact }}">{{ 'asp.' + workgroup.contact }}</a>
+        <script TYPE="text/javascript">
+            var mail_tag_id = '{{ ''.join( workgroup.name.split(' ') ) }}';
+            var mail_tag_element = document.getElementById( mail_tag_id );
+            mail_tag_element.innerHTML = mail_tag_element.innerHTML.split(".").splice(1,).join(".");
```

Comment:
A bit simpler:

```suggestion
            mail_tag_element.innerHTML = mail_tag_element.innerHTML.replace(/^asp./, "");
```

Reply:
The only risk I see in this is that the regex may give a different answer if ```.``` is removed from our prefix or it picks up string which start with ```asp``` for e.g. :
![image](https://user-images.githubusercontent.com/16386226/219480184-249d1103-a1c7-455c-96de-a3960d2bffd5.png)

but tbh this works in our case and any problem is unlikely to occur with this because our prefix will always have a ```.```. I am fine with this change as well.